Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -153,6 +153,46 @@ to create a Kubernetes cluster locally or using a managed Kubernetes service:
           - It might be necessary to add ``--host-dns-resolver=false`` if using the Virtualbox provider,
             otherwise DNS resolution may not work after Cilium installation.
 
+    .. group-tab:: Kubespray
+
+       `Kubespray <https://github.com/kubernetes-sigs/kubespray>`_ requires Python ≥ 3.10 for recent versions. 
+       For environment setup and dependencies installation, see the `Kubespray Ansible documentation <https://github.com/kubernetes-sigs/kubespray/blob/master/docs/ansible/ansible.md#installing-ansible>`_.
+
+       **Configure cluster:**
+
+       .. code-block:: bash
+
+           # Enter the already cloned kubespray directory
+           cd kubespray/
+           
+           # Copy sample inventory
+           cp -rfp inventory/sample inventory/mycluster
+           
+           # Configure your inventory
+           vi inventory/mycluster/inventory.ini
+           
+           # Configure Kubernetes networking:
+           # Use CNI without any network plugin
+           sed -e 's/^kube_network_plugin:.*$/kube_network_plugin: cni/' \
+               -e 's/^kube_owner:.*$/kube_owner: root/' \
+               inventory/mycluster/group_vars/k8s_cluster/k8s-cluster.yml > k8s-cluster.tmp
+           mv k8s-cluster.tmp inventory/mycluster/group_vars/k8s_cluster/k8s-cluster.yml
+
+       Setting ``kube_network_plugin: cni`` ensures the cluster deploys without any network plugin, allowing Cilium to be installed separately afterward.
+
+       **Deploy cluster:**
+
+       .. code-block:: bash
+
+           ansible-playbook -i inventory/mycluster/inventory.ini cluster.yml -b -v \
+             --private-key=~/.ssh/private_key
+
+           (Adjust the path to your private SSH key.)
```

Comment:
The note should not be part of the code block.

```suggestion
       .. code-block:: bash

           ansible-playbook -i inventory/mycluster/inventory.ini cluster.yml -b -v \
             --private-key=~/.ssh/private_key

       (Adjust the path to your private SSH key.)
```

Looks good otherwise.